helpers/bump-global-version.sh

```diff
@@ -0,0 +1,27 @@
+#! /bin/sh
+
+VERSION=$(grep '#*Lucet version ' Cargo.toml | sed 's/^ *# Lucet version *//')
+[ -z "$VERSION" ] && echo "Version header not found in the top Cargo.toml file" >&2 && exit 1
+
+dry_run() {
+    echo "Checking if the package can be published (dry run)..."
+    echo
+    find lucetc lucet-* benchmarks/lucet-* -type f -maxdepth 1 -name 'Cargo.toml' -print | while read -r file; do
+        dir="$(dirname $file)"
+        echo "* Checking [$dir]"
+        (cd "$dir" && cargo publish --allow-dirty --dry-run >/dev/null) || exit 1
+    done || exit 1
+    echo
+    echo "Done."
+}
+
+version_bump() {
+    echo
+    echo "Setting the global Lucet version number to: [$VERSION]"
+    find lucetc lucet-* benchmarks/lucet-* -type f -maxdepth 1 -name 'Cargo.toml' -print | while read -r file; do
+        sed -i '' "s/^ *version *=.*/version = \"${VERSION}\"/" "$file"
```

Comment:
the set of single quotes `''` after `-i` break it on my machine - sed then thinks the regular expression is the filename

Reply:
Updated :)